src/Map.js

```diff
@@ -527,15 +528,6 @@ const basemaps = {
     attribution:
       "Tiles &copy; Esri &mdash; Esri, DeLorme, NAVTEQ, TomTom, Intermap, iPC, USGS, FAO, NPS, NRCAN, GeoBase, Kadaster NL, Ordnance Survey, Esri Japan, METI, Esri China (Hong Kong), and the GIS User Community",
   },
-  // "Stadia.AlidadeSmooth": {
```

Comment:
Thank you for removing these 🙏 We ran into some server errors using these basemaps and I don't think we ever decided to add them back in.